### DISCUSSION TITLE
Using DEX as AD Login for ArgoCD - failed to create new access token

### DISCUSSION THREAD

**Discussion by teonius** [dexidp/dex#3128] [Q&A]
Hi,

we are trying to use DEX as AD Auth for ArgoCD, and we keep getting the following error message:

```
time="2023-09-28T06:54:01Z" level=error msg="failed to marshal offline session ID: string field contains invalid UTF-8"
time="2023-09-28T06:54:01Z" level=error msg="failed to create new access token: failed to marshal offline session ID: string field contains invalid UTF-8"
```
This comes when an AD User tries to log in. In the logs from ArgoCD we see the User data including groups hes a memeber of being correctly shown, but the login doesnt work with the above two error messages and the following error shown on the login page:

`failed to get token: oauth2: "server_error"`

has anyone encounter this before and has an idea what the issue is? We are stumbling in the dark at this point..
Thanks

**Comment by Szymen** [comment]
I'm facing the same issue! : ) No resolution yet.

**Comment by dragenet** [comment]
I faced the same issue and the problem was idAttr mapped to objectGUID  that was string like `4Fu1OvuLlemdVSmhDyYLag==`. I changed it to sAMAccountName and it works correctly
